pytensor/tensor/rewriting/subtensor.py

```diff
@@ -1445,7 +1445,7 @@ def local_adv_sub1_adv_inc_sub1(fgraph, node):
     if not isinstance(node.op, AdvancedSubtensor1):
         return
     inp = node.inputs[0]
-    if not inp.owner or not isinstance(inp.owner.op, AdvancedIncSubtensor1):
+    if inp.owner is None or not isinstance(inp.owner.op, AdvancedIncSubtensor1):
```

Comment:
This pattern I like to switch into `if not (inp.owner and isinstance(inp.owner.op, AdvancedIncSubtensor1)`. It explicitly states you are looking for an input that's an AdvancedIncSubtensor, and getting out if that's not the case.

I tried to match some of these cases with regex (maybe only some files), in one of the commits.